chatbot/test/fix-arg-qr-translation-test.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+const assert = require('assert');
+const modify = require('../insert/fix_arg_qr_translation.js');
+
+function readInputFile(filePath) {
+    return JSON.parse(fs.readFileSync(filePath).toString());
+}
+
+describe('Test of "Fixer Arg and QR after translation" function', function() {
```

Comment:
I'm not a fan of this "Test of 'x' function" boilerplate. I think you can just keep the "x" bit and drop everything else.